start.go

```diff
@@ -82,10 +82,8 @@ func start(c *cli.Context) error {
 		case err := <-done:
 			if err != nil {
 				log.Println("error: processed finished with error", err)
+				return err
```

Comment:
if the container exits cleanly this will just hang (why, is less clear), I think we should move this back